packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx

```diff
@@ -33,18 +35,13 @@ export default class BookReaderWrapper extends Component {
       searchInsideUrl: '/fulltext/inside.php',
       initialSearchTerm: null,
       imagesBaseURL: '/bookreader/BookReader/images/',
-      getPageURI: (index, reduce, rotate) => {
-        if ('undefined' == typeof(reduce))
-          reduce = 1;
-        if ('undefined' == typeof(rotate))
-          rotate = 0;
-        let uri = BookReader.prototype.getPageURI.call(this, index, reduce, rotate);
-        uri = uri + (uri.indexOf('?') > -1 ? '&' : '?');
-        uri = uri + 'scale=' + reduce + '&rotate=' + rotate;
+      getPageURI: (index, reduce = 1, rotate = 0) => {
+        let uri = originalGetPageURI.call(br, index, reduce, rotate);
```

Comment:
Question: Why use `br` instead of `this`?

Reply:
`this` was commandeered by the React component once the function was called.  The actual instance we want in this case is the new BookReader instance which is tacked to the window as `window.br`.